app/stv/stv.js

```diff
@@ -0,0 +1,353 @@
+      const minAlternatives = alternatives.filter(
+        (alternative) =>
+          (counts[alternative.description] || 0) <= minScore + EPSILON
+      );
+
+      // There is a tie for eliminating candidates. Per Scottish STV we must look at the previous rounds
+      if (minAlternatives.length > 1) {
+        let reverseIteration = iteration;
+
+        // Log the Tie
+        log.push({
+          action: 'TIE',
+          description: `There are ${
+            minAlternatives.length
+          } candidates with a score of ${Number(
+            minScore.toFixed(4)
+          )} at iteration ${reverseIteration}`,
+        });
+
+        // As long as the reveseindex is still larger then 1 we can look further back
+        while (reverseIteration >= 1) {
+          reverseIteration--;
+
+          // Find the log object for the last iteration
+          const logObject = log.find(
+            (entry) => entry.iteration === reverseIteration
+          );
+
+          // Find the lowest score (with regard to the alternatives in the actual iteration)
+          const iterationMinScore = Math.min(
+            ...minAlternatives.map((a) => logObject.counts[a.description] || 0)
+          );
+
+          // Find the candidates (in regard to the actual iteration) that has the lowest score
+          const iterationMinAlternatives = minAlternatives.filter(
+            (alternative) =>
+              (logObject.counts[alternative.description] || 0) <=
+              iterationMinScore + EPSILON
+          );
+
+          // If we are at iteration lvl 1 and there is still a tie we cannot do anything
+          if (reverseIteration === 1 && iterationMinAlternatives.length > 1) {
+            log.push({
+              action: 'TIE',
+              description:
+                'The backward checking went to iteration 1 without breaking the tie',
+            });
+            // Eliminate all candidates that are in the last iterationMinAlternatives
+            iterationMinAlternatives.forEach((alternative) => {
+              // Log the tie elimination
+              log.push({
+                action: 'TIEELIMINATE',
+                alternative: alternative,
+                minScore: Number(minScore.toFixed(4)),
+              });
+              doneAlternatives[alternative._id] = {};
+            });
+            break;
+          }
+
+          // If there is a tie at this iteration as well we must continue the loop
+          if (iterationMinAlternatives.length > 1) continue;
+
+          // There is only one candidate with the lowest score
+          const minAlternative = iterationMinAlternatives[0];
+          if (minAlternative) {
+            log.push({
+              action: 'ELIMINATE',
+              alternative: minAlternative,
+              minScore: Number(iterationMinScore.toFixed(4)),
+            });
+            doneAlternatives[minAlternative._id] = {};
+          }
+          break;
+        }
```

Comment:
@JonasBak This is how I did it. I hope we're on the same page now 📄 It simply eliminates candidates on unbreakable ties and logs it as a special event `TIEELIMINATE`.